test_fms/data_override/test_data_override2.sh

```diff
@@ -22,3 +22,10 @@ tnum=$( printf "%2.2d" 2 )
 sed "s/<test_num>/${tnum}/"  $top_srcdir/test_fms/data_override/input_base.nml > input.nml
 run_test test_data_override 2 skip
 
+tnum=$( printf "%2.2d" 3 )
+sed "s/<test_num>/${tnum}/"  $top_srcdir/test_fms/data_override/input_base.nml > input.nml
+srun -n 1 ./test_data_override
+
+tnum=$( printf "%2.2d" 4 )
+sed "s/<test_num>/${tnum}/"  $top_srcdir/test_fms/data_override/input_base.nml > input.nml
+srun -n 1 ./test_data_override_3D
```

Comment:
This should be `run_test` instead of `srun`

Reply:
fixed, also added skip because the appropriate INPUT files aren't in the CI